tests/test_models.py

```diff
@@ -82,14 +82,20 @@ def test_model_singlepsr_noise_dip_cusp(nodmx_psrs,caplog):
     assert hasattr(m,'get_lnlikelihood')
     x0 = {pname:p.sample() for pname,p in zip(m.param_names, m.params)}
     m.get_lnlikelihood(x0)
-
+    
 def test_model_singlepsr_noise_chrom_nondiag(nodmx_psrs,caplog):
     # caplog.set_level(logging.CRITICAL)
     m=models.model_singlepsr_noise(nodmx_psrs[1], chrom_gp=True)
     assert hasattr(m,'get_lnlikelihood')
     x0 = {pname:p.sample() for pname,p in zip(m.param_names, m.params)}
     m.get_lnlikelihood(x0)
-
+    m=models.model_singlepsr_noise(nodmx_psrs[1], chrom_gp=True,
+                                   chrom_gp_kernel='nondiag',
+                                   chrom_kernel='chrom')
```

Comment:
I don't think this actually tests the model we want.  There is no chromatic kernel called `'chrom'`.  This list of available models is `['periodic', 'sq_exp','periodic_rfband', 'sq_exp_rfband']`.  Which model do we want to test?